modules/apps/portal-store/portal-store-azure/src/main/java/com/liferay/portal/store/azure/AzureStore.java

```diff
@@ -208,14 +208,13 @@ public String[] getFileNames(
 		PagedIterable<BlobItem> pagedIterable = _blobContainerClient.listBlobs(
 			listBlobsOptions, null);
 
-		Stream<BlobItem> stream = pagedIterable.stream();
+		List<String> fileNames = new ArrayList<>();
```

Comment:
@jeffwu0724 Please double check the SF in https://github.com/brianchandotcom/liferay-portal/pull/130838, if Brian accepts it, please apply the same pattern here.